File: src/main/java/com/naminhyeok/fantazzk/CorsProperties.java
package com.naminhyeok.fantazzk;

import java.util.List;
import lombok.Getter;
import lombok.Setter;
import org.springframework.boot.context.properties.ConfigurationProperties;

@ConfigurationProperties(prefix = "cors")
@Getter
@Setter
public class CorsProperties {
    private List<String> allowedOrigins = List.of("http://localhost:3000", "http://localhost:5173");
}

File: src/main/kotlin/com/naminhyeok/fantazzk/CorsProperties.kt
package com.naminhyeok.fantazzk

import org.springframework.boot.context.properties.ConfigurationProperties

@ConfigurationProperties(prefix = "cors")
data class CorsProperties(
    val allowedOrigins: List<String> = listOf("http://localhost:3000", "http://localhost:5173"),
)
